app/src/ui/move-to-applications-folder.tsx

```diff
@@ -57,13 +57,14 @@ export class MoveToApplicationsFolder extends React.Component<
     )
   }
 
-  private onSubmit = async () => {
+  private onSubmit = () => {
+    this.props.onDismissed()
+
     try {
       this.props.dispatcher.moveToApplicationsFolder()
```

Comment:
You can replace this line with `throw new Error('some message')` if you want to test it 😄